BaseApp/COD3023.txt

```diff
@@ -0,0 +1,39 @@
+OBJECT Codeunit 3023 DotNet_Uri
+{
+  OBJECT-PROPERTIES
+  {
+    Date=04-05-18;
+    Time=09:10:02;
+    Modified=Yes;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    OnRun=BEGIN
+          END;
+
+  }
+  CODE
+  {
+    VAR
+      DotNetUri@1000 : DotNet "'System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.Uri";
```

Comment:
No necessary to specify a version. But it does not harm as such. But this would suffice:
DotNetUri@1000 : DotNet "'System'.System.Uri";

Reply:
Actually, in all other dotnet wrapper codeunits using system.dll, the version is specified as well. It doesn't work without.